packages/big-design/src/components/Dropdown/Dropdown.tsx

```diff
@@ -94,23 +106,29 @@ export class Dropdown<T extends any> extends React.PureComponent<DropdownProps<T
           >
             {option.type === 'link' && !option.disabled ? (
               <Link href={option.url} target={option.target}>
-                <Flex alignItems="center" flexDirection="row">
-                  {icon && <FlexItem paddingRight="xSmall">{this.renderIcon(option, isHighlighted)}</FlexItem>}
-                  {content}
-                </Flex>
+                {listItemContent}
               </Link>
             ) : (
-              <Flex alignItems="center" flexDirection="row">
-                {icon && <FlexItem paddingRight="xSmall">{this.renderIcon(option, isHighlighted)}</FlexItem>}
-                {content}
-              </Flex>
+              listItemContent
             )}
           </ListItem>
         );
       })
     );
   }
 
+  private wrapInTooltip(element: ReactElement, message: string, placement: PopperProps['placement'] = 'bottom') {
+    return (
+      <Tooltip
```

Comment:
We may want to the tooltip in this case to be `display: block;` so the tooltip is not right up on the text.